Cargo.toml

```diff
@@ -36,3 +36,8 @@ tokio = { version = "1.44.2", features = ["full"] }
 clap = { version = "4.5.37", features = ["derive"] }
 arrow = { version = "57.0.0", features = ["ipc_compression"] }
 chrono = { version = "0.4", features = ["clock", "std", "wasmbind"] }
+
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+opendal = { version = "0.53", features = ["services-s3"] }
```

Comment:
Can it be 
opendal =  version = "0.53
directly?
Since we already enable the feature in `crates/fluss/Cargo.toml`

Reply:
done